src/bp/core/services/kvs/service.ts

```diff
@@ -0,0 +1,42 @@
+import * as sdk from 'botpress/sdk'
```

Comment:
Similarly to the ghost, I think we could put the Global version and Scoped version inside the same file (kvs.ts) so it's easier to find & work with, since one does simply change some parameters